Funcky/Sequence/Sequence.FromNullable.cs

```diff
@@ -6,11 +7,11 @@ public static partial class Sequence
     [Pure]
     public static IEnumerable<TResult> FromNullable<TResult>(TResult? element)
         where TResult : class
-        => element is null ? Enumerable.Empty<TResult>() : Return(element);
+        => element is null ? [] : Return(element);
```

Comment:
Why not also replace the `Return`?

Reply:
Addressed in 9ac24e9267e113684d68c0ac8a73ca2c27c9deb2